admin/views/paper-collapsible.php

```diff
@@ -20,23 +22,25 @@
 }
 
 ?>
-<div class="<?php echo esc_attr( 'paper tab-block ' . $class ); ?>"<?php echo ( $paper_id ) ? ' id="' . esc_attr( 'wpseo-' . $paper_id ) . '"' : ''; ?>>
+<div
+	class="<?php echo esc_attr( 'paper tab-block ' . $class ); ?>"<?php echo ( $paper_id ) ? ' id="' . esc_attr( $paper_id_prefix . $paper_id ) . '"' : ''; ?>>
 
 	<?php
 	if ( ! empty( $title ) ) {
 		if ( ! empty( $collapsible ) ) {
 
 			$button_id_attr = '';
 			if ( ! empty( $paper_id ) ) {
-				$button_id_attr = sprintf( ' id="%s"', esc_attr( 'wpseo-' . $paper_id . '-button' ) );
+				$button_id_attr = sprintf( ' id="%s"', esc_attr( $paper_id_prefix . $paper_id . '-button' ) );
 			}
 
 			printf(
-				'<h2 class="collapsible-header"><button%4$s type="button" class="toggleable-container-trigger" aria-expanded="%3$s">%1$s <span class="toggleable-container-icon dashicons %2$s" aria-hidden="true"></span></button></h2>',
+				'<h2 class="collapsible-header %5$s"><button%4$s type="button" class="toggleable-container-trigger" aria-expanded="%3$s">%1$s <span class="toggleable-container-icon dashicons %2$s" aria-hidden="true"></span></button></h2>',
```

Comment:
Maybe also add an ignore for the output escaping and make sure all input variables are properly escaped.

Reply:
Added `esc_attr` around the added attribute. Leaving the others as you thought there is a PR for that.